aws-mfa

```diff
@@ -273,8 +297,8 @@ def get_credentials(short_term_name, lt_key_id, lt_access_key, args, config):
     with open(AWS_CREDS_PATH, 'w') as configfile:
         config.write(configfile)
     logger.info(
-        "Success! Your credentials will expire in %s seconds at: %s"
-        % (args.duration, response['Credentials']['Expiration']))
+        "Success! Your credentials will expire at: %s" %
+        response['Credentials']['Expiration'])
```

Comment:
If we let AWS define a default duration, then we need to do some time differential...
(Now - response['Credentials']['Expiration'])
Which is more messy, so I just removed it. We could add it back